types/node/index.d.ts

```diff
@@ -2732,6 +2732,9 @@ declare module "child_process" {
         stdout: stream.Readable;
         stderr: stream.Readable;
         stdio: [stream.Writable, stream.Readable, stream.Readable];
+        spawnargs: [string];
+        spawnfile: string;
+        connected: string;
```

Comment:
Should `connected` be `boolean`?

Reply:
connected is already present as boolean at line 2744